batcher/aligned-batcher/src/halo2/ipa/mod.rs

```diff
@@ -21,6 +22,17 @@ pub const MAX_VERIFIER_KEY_SIZE: usize = 1024;
 pub const MAX_IPA_PARAMS_SIZE: usize = 4 * 1024;
 
 pub fn verify_halo2_ipa(proof: &[u8], public_input: &[u8], verification_key: &[u8]) -> bool {
+    if verification_key.len() < 12 {
```

Comment:
where does this restriction (`< 12`) come from? there should be some comment justifying it. Could this change in the future and make things break?

Reply:
nvm, I understood it following the PR, but this should be documented somewhere. Does `verification_key` have always a fixed size of 12 for IPA halo2?

Reply:
The verification key contains additional data alongside the verification key needed to verify the Halo2 proof. Specifically the constrain system, and verification parameters. I added comments outlining the serialization format ref: https://github.com/yetanotherco/aligned_layer/pull/738/files#diff-66d2c3bc70bb4a1f5bfb8ca204b4a625aef017836a07ff3f5c9c1adf159dba0bR17